src/components/Table/TableFormatters/AccountConnectFormatter.vue

```diff
@@ -115,8 +115,7 @@ export default {
         const url = this.formatterArgs.assetUrl.replace('{id}', assetId)
         const res = await this.$axios.get(url)
 
-        // TODO 暂时将 sftp 隐藏
-        this.protocols = res.protocols.filter(protocol => (protocol.name !== 'sftp') && (protocol.name !== 'winrm')) || []
+        this.protocols = res.protocols.filter(protocol => (protocol.name !== 'winrm')) || []
       } catch (e) {
         throw new Error(`Error getting protocols: ${e}`)
       }
```

Comment:
The first difference between two versions of the code is that we now have "export" keyword instead of the previous import statement which makes it difficult to use other files.

For potential issues:

It seems like there might be some errors in the JavaScript syntax used here but not clearly mentioned. There should probably no need to add `@` before the line number because you already specified it as global level scope.

Also, please make sure these changes do not interfere with existing dependencies/usage in any way!